trucks.js

```diff
@@ -504,8 +504,68 @@ var foodTrucks = [
 ];
 
 // this module should support the following methods:
+
+var _ = require('underscore');
+
 // getTrucks() - return all trucks
+function getTrucks() {
+
+	return foodTrucks;
+}
+
 // getTruck(name) - return the truck object matching 'name'
+function getTruck(name) {
+
+	var match = _.filter(foodTrucks, function(truck){ 
+
+		// return _.contains(truck.name, a);
+		if (truck.name === name) {
+			return truck;
```

Comment:
The function you pass to _.filter should technically return true or false. This works because you're returning either an object or undefined under the same circumstances, but just to be clear about how _.filter works.